src/protocols/rdp/channels/audio-input/audio-buffer.h

```diff
@@ -66,6 +66,27 @@ typedef struct guac_rdp_audio_format {
 
 } guac_rdp_audio_format;
 
+/**
+ * A structure of list to take the audio buffers.
+ */
+typedef struct audio_stream_list {
+    /**
+     * The data of audio buffer
+     */
+    char*  data;
+
+    /**
+     * The length of audio buffer
+     */
+    int    length;
+
+    /**
+     * The pointer of the next audio buffer
+     */
+    void*  next;
```

Comment:
This should be a pointer to a concrete type, presumably `audio_stream_list` (which would then need a forward declaration). We should only be using `void*` for pointers that truly are intended to point to arbitrary data.